cfme/tests/infrastructure/test_provisioning.py

```diff
@@ -40,6 +40,33 @@ def vm_name():
     return vm_name
 
 
+@pytest.fixture(scope="function")
+def provision_request(provisioning, provider, request, vm_name):
+    # generate_tests makes sure these have values
+    template, host, datastore = map(provisioning.get, ('template', 'host', 'datastore'))
```

Comment:
OPTIONAL: I'd recommend you to use list comprenesion instead of functional programming things